<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dash</class>
 <widget class="QDialog" name="Dash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1910</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>230</y>
      <width>511</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Rounded</family>
      <pointsize>28</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: transparent;
font: 81 28pt &quot;SF Pro Rounded&quot;;</string>
    </property>
    <property name="text">
     <string>Face Analysis &amp; ANPR</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameF_2">
    <property name="geometry">
     <rect>
      <x>1710</x>
      <y>370</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frameP_2">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>370</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Rounded</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Robot</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>1111</width>
      <height>661</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 126, 249);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="image_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>63</width>
      <height>63</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{

background-color: rgb(59, 103, 128);
border-radius: 7px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="start_stop_btn">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/icons/run_video.png</normaloff>:/icons/icons/run_video.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="borrar_ok_3">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>920</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Rounded</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(74, 37, 0);
background-color: rgb(221, 165, 51);</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>10</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Rounded</family>
      <pointsize>28</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">

background-color: transparent;
font: 81 28pt &quot;SF Pro Rounded&quot;;</string>
    </property>
    <property name="text">
     <string>Remote the robot</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_employees">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>730</y>
      <width>901</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Display</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>EmployeeID</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>191</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>FirstName</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color>
       <red>85</red>
       <green>255</green>
       <blue>127</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LastName</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>248</green>
       <blue>53</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>License Number Plate</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DepartmentID</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DOE</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>840</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bt_add">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(221, 165, 51);
color: rgb(74, 37, 0);

font: 81 10pt &quot;SF Pro Rounded&quot;;
</string>
       </property>
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_decline">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(221, 165, 51);
color: rgb(74, 37, 0);

font: 81 10pt &quot;SF Pro Rounded&quot;;
</string>
       </property>
       <property name="text">
        <string>DECLINE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_viewall">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(221, 165, 51);
color: rgb(74, 37, 0);

font: 81 10pt &quot;SF Pro Rounded&quot;;
</string>
       </property>
       <property name="text">
        <string>VIEW ALL DATA</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>500</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Details</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>139</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Name : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Status :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Clocked Time : </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>131</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="NameLabel">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="StatusLabel">
        <property name="font">
         <font>
          <family>Roboto Condensed Light</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="HoursLabel">
          <property name="font">
           <font>
            <family>Roboto Condensed Light</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="MinLabel">
          <property name="font">
           <font>
            <family>Roboto Condensed Light</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_4">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>140</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>◀️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_6">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>140</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>▶️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_8">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>90</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>🔼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_2">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>190</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>🔽</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_7">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>80</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>↖️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_9">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>80</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>↗️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_1">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>200</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>↙️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_3">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>200</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>↘️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_0">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>140</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 37, 0);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>🔄</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>221</width>
      <height>106</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="2" column="1">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1">
        <widget class="QLabel" name="time_label_2">
         <property name="font">
          <font>
           <family>SF Pro Rounded</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
background-color: transparent;
</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>SF Pro Rounded</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: transparent;
</string>
       </property>
       <property name="text">
        <string>Date :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="date_label_2">
       <property name="font">
        <font>
         <family>SF Pro Rounded</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: transparent;
</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>SF Pro Rounded</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: transparent;
</string>
       </property>
       <property name="text">
        <string>Time :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
